README.md

```diff
@@ -15,6 +15,11 @@ cd picasclient
 pip install -U .
```

Comment:
Clone using the https web URL instead of git for the generic user access to the repo